go.mod

```diff
@@ -295,7 +295,7 @@ replace (
 	cosmossdk.io/core v0.12.0 => cosmossdk.io/core v0.11.0
 
 	// Direct cosmos-sdk branch link: https://github.com/piplabs/cosmos-sdk/tree/piplabs/v0.50.7, current branch: piplabs/v0.50.7
-	github.com/cosmos/cosmos-sdk => github.com/piplabs/cosmos-sdk v0.50.7-piplabs-v0.12
+	github.com/cosmos/cosmos-sdk => github.com/ezreal1997/cosmos-sdk v0.50.7-test-v0.2
```

Comment:
Let's change this after merging https://github.com/piplabs/cosmos-sdk/pull/16 and release.